### ISSUE TITLE
Ikaruga on Retroarch visual issue

### ISSUE THREAD

**Issue by Josaem** [flyinghead/flycast#592]
System: Retroarch Android 32bit
Game: Ikaruga
Issue: Some bullets show a circle around them, similar to its hitbox
Additional Info: This issue seems to be appearing on the latest version of the Flycast core, although not in the app. I have tried using both default settings for Retroarch and the core, using vulkan and changing most settings, such as the resolution, etc. The issue is still present after trying those.
The 64 bit version also has this issue, although Retroarch Plus doesn't. 

![Screenshot_20220411_165424_com retroarch](https://user-images.githubusercontent.com/82065172/162778943-b060a3a8-c6fa-4525-8ca5-1d5d39e41088.jpg)

**Comment by flyinghead** [comment]
Can you post the version(s) of the Flycast core in the various RetroArch apps?
Some RA builds don't have a core updater so they might be running older versions.

Also, some logs (both for the working RA and the non-working ones, including core logging) would be useful to troubleshoot the problem.

**Comment by Josaem** [comment]
The version I'm running on Retroarch Plus is Flycast (0.1 4a913e0), while the one I'm running on Retroarch (32 bit, downloaded from the Retroarch website) is Flycast (v1.3). So this means that the issue is caused by the recent update, which makes sense since I started getting the issue after updating the core a few days ago!

**Comment by flyinghead** [comment]
Version 0.1 is very old.

Enable Log to File in the settings and you'll find a `retroarch.log` file in `RetroArch/logs`

Also, please indicate which platform you're using.

**Comment by Josaem** [comment]
Okay, do I have to open the game a run it until I see the visual bug? Or should I just open it?

As for the platform I'm running Ikaruga through Retroarch on android (Huawei Mate 20). I'll post the log files indicating which one corresponds to each version of Retroarch.

**Comment by flyinghead** [comment]
I'm more interested in the initialization log, so just starting the game should be enough.

**Comment by Josaem** [comment]
Here you go. The retroarch.log file corresponds to Retroarch 1.10.2 while retroarchPlus.log corresponds to Retroarch Plus 1.9.12.

https://drive.google.com/file/d/1lUPdkKbaNbxSr0-0gNlL6w30FD6yj9Bx/view?usp=drivesdk

https://drive.google.com/file/d/1lYFVyYqvtwNcmPqud_5S8eXt3QoTqPB6/view?usp=drivesdk

**Comment by flyinghead** [comment]
Unfortunately nothing suspicious is revealed in these logs.

**Comment by flyinghead** [comment]
Do you also have the issue with the standalone Flycast app?

**Comment by Josaem** [comment]
Nope, the issue only occurs when I use the core, although I get some weird stuttering when I use the app.

**Comment by gouchi** [comment]
@Josaem May you try to make a test with [RetroArch](https://f-droid.org/en/packages/com.retroarch/) from [F-Droid](https://f-droid.org) ?

Thank you.
